library/CloudBlobHelpers.cs

```diff
@@ -134,7 +134,8 @@ public static async Task<ReplicatedTableReadBlobResult> TryReadBlobAsync<T>(Clou
         {
             try
             {
-                string content = await blob.DownloadTextAsync(ct);
+                BlobRequestOptions options = new BlobRequestOptions() { ServerTimeout = TimeSpan.FromSeconds(30) };
+                string content = await blob.DownloadTextAsync(null, null, options, null, ct);
```

Comment:
Do we handle timeout failures?

Reply:
We try to download 3 times(was 2 before this change) in the life cycle of the configuration to make sure it does not become stale.